scripts/upgrade

```diff
@@ -76,15 +23,13 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading source files..." --weight=4
 
-	ynh_secure_remove --file=$final_path
-
+	ynh_secure_remove --file=$install_dir
```

Comment:
There's an inconsistency with install script for which the folder already exists ... so let's mkdir it:

```
mkdir "$install_dir"
```